tinygrad/codegen/search.py

```diff
@@ -3,8 +3,11 @@
 from tinygrad.codegen.linearizer import Linearizer
 from tinygrad.helpers import DEBUG, prod, getenv
 
-UPCASTS = [1,2,3,4,5,6,7,8]
-LOCALS = [1,2,3,4,5,6,7,8,16,24,32]
+def get_divisors(n, min_div = 1, max_div = 512):
+  if min_div > 1: yield 1
+  for d in range(min_div, max_div + 1):
```

Comment:
Dividing by zero gives error and divide by one doesn't make sense?
(Removing line 9 too)
```suggestion
  assert min_div > 0
  for d in range(min_div, max_div + 1):
```